kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeExampleDataTab.js

```diff
@@ -216,45 +246,76 @@ define ([
+                            };
+                            if ($(this).text().split(" ")[1] === "Copy"){
+                                var dialog = new BootstrapDialog({
+                                    title: 'Item already exists in workspace under same name.',
+                                    body: 'Do you want to override existing copy?',
+                                    buttons: [$('<a type="button" class="btn btn-default">')
+                                        .append('Yes')
+                                        .click(function () {
+                                            dialog.hide();
+                                            updateButton.call(this);
+
+                                        }.bind(this))
+                                        , $('<a type="button" class="btn btn-default">')
+                                        .append('No')
+                                        .click(function () {
+                                            dialog.hide();
 
-                            var thisBtn = this;
-                            Promise.resolve(self.serviceClient.sync_call(
-                                "NarrativeService.copy_object",
-                                [{
-                                    ref: object_info[6] + "/" + object_info[0],
-                                    target_ws_name: self.narWs,
-                                }]
-                            ))
-                            .then(function(info) {
-                                $(thisBtn).html('Added');
-                                self.trigger('updateDataList.Narrative');
-                            })
-                            .catch(function(error) {
-                                $(thisBtn).html('Error');
-                                if (error.error && error.error.message) {
-                                    if (error.error.message.indexOf('may not write to workspace')>=0) {
-                                        self.options.$importStatus.html($('<div>').css({'color':'#F44336','width':'500px'}).append('Error: you do not have permission to add data to this Narrative.'));
-                                    } else {
-                                        self.options.$importStatus.html($('<div>').css({'color':'#F44336','width':'500px'}).append('Error: '+error.error.message));
-                                    }
-                                } else {
-                                    self.options.$importStatus.html($('<div>').css({'color':'#F44336','width':'500px'}).append('Unknown error!'));
-                                }
-                                console.error(error);
-                            });
+                                        })
+                                    ],
+                                    closeButton: true
+                                });
+                                dialog.show();
+                            }else{
```

Comment:
please add a space next to curly braces adjacent to control statements (if, else, and so on)